### PR TITLE
match with code in NotificationManager

### PR DESCRIPTION
**Pull request by cjakeman**
Many changes to fit with code developments

### CODE REVIEW

File: menu.json [hunk 1]
```diff
@@ -1,249 +1,232 @@
...
-    ],
-    "checkList": [
+      ],
+      "metLists": {
+        "itemList": [
+          {
+            "$type":"ORTS.Link, Menu",
+            "label": "What's new",
+            "value": "Find out on-line what's new in this version.",
+            "Url": "https://www.openrails.org/discover/latest_stable_version",
```
Comment (twpol) [inline menu.json]: Did this key want to be `url` (all lower-case) as well?

**Review by twpol — DISMISSED** [review]
All good with me, though we'll need to update the schema here or later to match.

File: menu.json (hunk 1 above)
Comment (cjakeman) [reply menu.json]: Eagle-eyes !

Corrected. Thanks, James.

File: schema.json [hunk 2]
```diff
@@ -1,236 +1,293 @@
...
-                    "updateChannel": {
-                        "type": "string",
-                        "enum": ["stable", "testing", "unstable"]
+                    "url": {
+                      "type": "string"
                     },
-                    "prefixItemList": {
-                        "type": "array",
-                        "items": {
-                            "$ref": "#/definitions/item"
```
Comment (twpol) [inline schema.json:33]: This schema looks a lot less strict than mine; perhaps we can address that in a later update? For example, these items share a common definition which uses `oneOf` to ensure each type of item has the right properties.

**Review by twpol — DISMISSED** [review]

**Review by twpol — APPROVED** [review]

**Pull request merged**